### PR TITLE
Create storybook stories to show different font types

### LINKED ISSUE #1936: Create storybook stories to show different font types
**Is your feature request related to a problem? Please describe.**
Currently, we don't have a way of visualizing the different fonts types we use.

**Describe the solution you'd like**
Create a story in storybook `utilities/psammead-styles` to show text rendered in the different font types defined in https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.js

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #1936 

**Overall change:** _Create a story in storybook to show different font types_

**Code changes:**

- _Add a `font styles` story under `utilities/psammead-styles`._
- _Add `json5` npm package to parse unquoted text to JSON object._
- _Show Pangram rendered in different font types in storybook._

**Screenshot**
Screen Shot 2019-09-02 at 1 25 29 PM

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by sareh — CHANGES_REQUESTED** [review]
@EinsteinCarrey Looks good so far! I like the drop-down that selects and applies the different fonts. I think the example text needs to be updated. It currently doesn't have any bold text, so the bold font file examples aren't being applied. 
You can get example text with all of the english alphabet e.g. `The quick brown fox jumps over the lazy dog 0123456789`
Then equivalent pangrams for the different scripts would be useful. e.g. I found this one for Arabic text when searching [Reference](https://arabicadventure.blogspot.com/2009/04/arabic-panagram.html)
`نص حكيم له سر قاطع وذو شأن عظيم مكتوب على ثوب أخضر ومغلف بجلد أزرق`
(Which I'm told means this:
"A wise text which has an absolute secret and great importance, written on a green tissue and covered with blue leather")

So we can have these, in spans and in bold tags:
`The quick brown fox jumps over the lazy dog 0123456789`
`١٢٣٤٥٦٧٨٩٠ نص حكيم له سر قاطع وذو شأن عظيم مكتوب على ثوب أخضر ومغلف بجلد أزرق`

**Review by RayNjeri — APPROVED** [review]
Good job.

File: packages/utilities/psammead-styles/src/index.stories.jsx [hunk 1]
```diff
@@ -51,9 +54,47 @@ const Detects = styled.li`
...
+const getFontStyles = fontName => {
+  const font = fontNames.find(x => x.includes(fontName.replace(/ /g, '_')));
+  const fontFace = fonts[font] || '';
+  const fontStyle = fontFace
+    .replace(/"/g, '')
+    .replace('@font-face', '')
+    .replace(/: /g, ':"')
+    .replace(/;/g, '",');
+
+  const { fontFamily, fontWeight } = json5.parse(camelCase(fontStyle));
```
Comment (dr3) [inline packages/utilities/psammead-styles/src/index.stories.jsx:74]: Why do you need `json5` here? howcome `JSON.parse` work? 🤔

Comment (EinsteinNjoroge) [reply packages/utilities/psammead-styles/src/index.stories.jsx:74]: `JSON.parse `does not work for unquoted values that's why we need the extended functionality of [Json5](https://json5.org/)

Comment (dr3) [reply packages/utilities/psammead-styles/src/index.stories.jsx:74]: Ahhhh ok, for others benefit `camelCase(fontStyle)` is 
```js
{
  fontFamily:"Iskoola Pota BBC",
  fontWeight:"700",
  fontStyle:"normal",
  src:"url('https://wsDownloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/bold.woff') format('woff'), url('https://wsDownloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/bold.eot') format('eot'), url('https://wsDownloads.files.bbci.co.uk/fonts/IskoolaPota/v5.91/bold.ttf') format('ttf')",
  fontDisplay:"swap",
}
```

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM, just needs package and changelog bump

**Review by dr3 — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Thanks for making the updates. Looks good!

**Pull request merged**